sites/public/src/components/listings/search/ListingsSearchCombined.tsx

```diff
@@ -230,6 +246,18 @@ const locations: FormOption[] = [
     label: "Sonoma",
     value: "Sonoma",
   },
+  {
+    label: "San Francisco",
+    value: "San Francisco",
+    isDisabled: true,
+    labelNoteHTML: `(For San Francisco listings, please go to <a href="https://housing.sfgov.org/">DAHLIA</a>)`,
```

Comment:
can we make this open in a different tab/window (ie target=_blank)?

Reply:
Resolving now so I can merge this in, I hope you don't mind. We can do this part next week. Thanks!!!